cmd/sink/filters/filters.go

```diff
@@ -141,15 +150,20 @@ func (f *Filters) createNamespaceFilters(namespace, file string, globalArchive,
 // deleting. createFilters wraps all errors it encounters while creating filters and returns the wrapped error. Even if
 // the returned error is not nil, the maps returned can still be used.
 func (f *Filters) createFilters(
-	namespace string,
-	file string,
-	globalArchive, globalDelete map[string]string,
-) (map[NamespaceGroupVersionKind]cel.Program, map[NamespaceGroupVersionKind]cel.Program, error) {
+	namespace, file string,
+	globalArchive, globalDelete, globalArchiveOnDelete map[string]string,
+) (
+	map[NamespaceGroupVersionKind]cel.Program,
+	map[NamespaceGroupVersionKind]cel.Program,
```

Comment:
Is there a blank line or something there?  No line number, but it looks strange.

Reply:
I'm not seeing the blank line that you see...

Reply:
Ignore it.